test/MorphoTokenEthereumTest.sol

```diff
@@ -65,6 +69,9 @@ contract MorphoTokenEthereumTest is BaseTest {
 
         deal(address(newMorpho), delegator, amount);
 
+        vm.expectEmit(address(newMorpho));
+        emit DelegationToken.DelegateeChanged(delegator, address(0), delegatee);
+        emit DelegationToken.DelegatedVotingPowerChanged(delegatee, 0, amount);
```

Comment:
This pattern does not work: the last line will just be emitted in the test and not be expected to be emitted by a call made later.
```suggestion
        vm.expectEmit(address(newMorpho));
        emit DelegationToken.DelegateeChanged(delegator, address(0), delegatee);
        vm.expectEmit(address(newMorpho));
        emit DelegationToken.DelegatedVotingPowerChanged(delegatee, 0, amount);
```